ng-dev/pr/merge/merge-tool.ts

```diff
@@ -316,4 +318,15 @@ export class MergeTool {
       Log.debug('Assuming correct access because this a bot account.');
     }
   }
+
+  parsePrNumber(prUrlOrNumber: number | string): number {
```

Comment:
Rather than parsing this information in the tool, we should do the extra parsing during the parsing phase of `yargs` tooling?  https://github.com/angular/dev-infra/blob/03b8a7dffd1205e061f0bee949024ebefc2a6592/ng-dev/pr/merge/cli.ts#L25

Because I believe that `yargs` will try to coerce whatever is provided here into a number so the `typeof` check would fail anyway

You will get this failure `Variable $number of type Int! was provided invalid value`

Reply:
Ah! I wondered if I was doing this in the wrong spot. I'll take a look.

Reply:
I've updated. Looks like you can't do multiple types on yargs. So I had to go with `string` and then parse from there.